.github/workflows/issue-write.yml

```diff
@@ -9,6 +9,7 @@ on:
       - "Code lint"
       - "CI Checks"
       - "Test Issue Write"
+      - "Check LLVM ABI annotations"
```

Comment:
@tstellar @boomanaiden154 
I would like to land this separately so we can test that the whole workflow is working as expected. Does the current name sound fine to you? If so, I'll create another PR with just these changes.

Reply:
Seems reasonable enough to me.

Reply:
Looks good to me too.